assets/src/scss/pages/post/_article-content.scss

```diff
@@ -299,7 +299,7 @@
   }
 
   img {
-    max-width: 100%;
+    width: 100%;
```

Comment:
This ends up adapting the image width to the caption width, which might generate unexpected layout changes. But I'm not sure how to handle that, there are several cases:

Testing with an image 150x150 to see the stretch

What I've inserted, disabling `width:100%` (it falls back to a `max-width: 100%;` from `div.page-template img` on a page; appears the same in the editor):
![Screenshot from 2021-03-12 10-12-08](https://user-images.githubusercontent.com/617346/110918514-8bef3f00-831b-11eb-8f15-3cbd245a5544.png)

Current PR:
On insertion
![Screenshot from 2021-03-12 10-06-37](https://user-images.githubusercontent.com/617346/110918469-7e39b980-831b-11eb-969a-88a7d16dfc4a.png)
When adding an alignement
![Screenshot from 2021-03-12 10-10-00](https://user-images.githubusercontent.com/617346/110918477-80037d00-831b-11eb-9293-1123f368f3c5.png)

(Weirdly, on insertion there is no alignement, and once you add one you can only change it, not remove it (but it's not part of this ticket))

My guess would be, we should probably not change anything on how images are currently streched but maybe there's a design choice there ? No idea of the impact it would have on current content

Reply:
That's a good point, it's not the scope of this ticket to fix this and it might cause issues. I'll revert the changes related to the width 👍